bb8/src/api.rs

```diff
@@ -45,6 +46,11 @@ impl<M: ManageConnection> Pool<M> {
         Builder::new()
     }
 
+    /// Returns statistics about the historical usage of the pool.
+    pub fn statistics(&self) -> Statistics {
```

Comment:
Hmm, actually I think we should make `statistics` a field in `State`. What do you think?

Reply:
I would expect that for having a full representation of all metrics, the ones coming from the `State` would end us gauge metrics, the user will need to gather both objects, so Im ok to make this easier to the user by only requiring to make a single api call for gathering both.

Ill do that.